tests/Tests/Framework/SerializationTests/ExceptionSerializationTests.cs

```diff
@@ -28,7 +28,8 @@ public ExceptionSerializationTests()
 			_elasticsearchNetSerializer = lowlevelClient.Serializer;
 		}
 
-		[U]
+		//TODO Needs exception formatter
+		[U(Skip="Needs Exception formatter")]
```

Comment:
should this be addressed before merging this in?

Reply:
Yep!

Reply:
Done!